sqldef.go

```diff
@@ -68,22 +69,32 @@ func Run(generatorMode schema.GeneratorMode, db database.Database, sqlParser dat
 	}
 }
 
-// TODO: Warn if both the second --file and database options are specified
-func ParseFiles(files []string) (string, string) {
+func ParseFiles(files []string) []string {
 	if len(files) == 0 {
 		panic("ParseFiles got empty files") // assume default:"-"
 	}
 
-	desiredFile := files[0]
-	currentFile := ""
-	if len(files) == 2 {
-		desiredFile = files[1]
-		currentFile = files[0]
-	} else if len(files) > 2 {
-		fmt.Printf("Expected only one or two --file options, but got: %v\n", files)
-		os.Exit(1)
+	result := make([]string, 0, len(files))
+	re := regexp.MustCompile("[, ]")
```

Comment:
Which are you going to actually use, `,` or ` `? If you're going to use only either of these, nobody might use the other, so I want to support only one of these.

Reply:
I'll use `,`! I'll delete ` `.